task-impls/src/da.rs

```diff
@@ -597,13 +597,20 @@ where
                     Ok(Ok(_)) => continue,
                 }
             }
-            let mut txns = vec![];
-            for (_hash, txn) in unclaimed_txns {
-                txns.push(txn.clone());
-            }
-            return Some(txns);
+            break;
```

Comment:
IMO it's clearer to put this `break` in the `Err(_)` case in the `match` above, to make it clear that this only happens in one scenario (timeout). That's mostly personal preference though.

Reply:
Agree! I put it there because there was some code after the `match`, but that's not the case anymore. Will move it.

Reply:
Oh actually I think the `break` should be put here, otherwise we aren't handling the case where the `if` conditions (line 580) aren't satisfied.

Reply:
Oh yeah you're right